teamplate.rb

```diff
@@ -18,11 +18,12 @@
 generate "bootstrap:install", "static"
 generate "bootstrap:layout" # auto overwrite
 
+# Figaro
 # Other gems
-%w( quiet_assets pg ).each { |name| gem name }
+%w( quiet_assets pg rails-erd figaro).each { |name| gem name }
```

Comment:
What would be the syntax if the command is "figaro install"? I tried to just add it as a string and also generate "figaro install", but neither worked